.pre-commit-config.yaml

```diff
@@ -1,6 +1,7 @@
 ci:
   autofix_prs: false
   autoupdate_schedule: 'monthly'
+  autoupdate_commit_msg: "Chore: update pre-commit hooks"
```

Comment:
This is error PC901: https://learn.scientific-python.org/development/guides/style/#pre-commit. The suggestion is that you probably want to have a more specific comment message rather than the one the bot defaults to.

Reply:
I can see why repositories that are just starting to use autoupdate might find it useful to customize the commit message, but `astropy` already has 79 commits that contain `[pre-commit.ci]` in their message (according to `git log --grep='\[pre-commit\.ci\]' --oneline | wc -l`). Having multiple patterns to grep for is not helpful.

Reply:
I would agree we wait on this for a discussion of e.g. [commitizen](https://commitizen-tools.github.io/commitizen/).
As a note to head off discussion, I'm not advocating for applying commitizen to all commits, only ones applied to `main`. Specifically, this would mean that a PR can have trash commit messages (Im as guilty as the rest) but when we squash-merge the single commit message must be excellent. If someone chooses to not squash-merge then they would likewise be required to have quality commit messages.